### ISSUE TITLE
Forked process runs indefinitely

### ISSUE THREAD

**Issue by MattBred** [elastic/apm-agent-php#627]
**Describe the bug**
If I have the APM extension enabled and fork a child, the forked child **never stops**.

**To Reproduce**
```php
<?php

$child = $pid = pcntl_fork();
if ($child === 0)
{
	sleep(1);
	echo "\n\n-----------------------CHILD EXITING-------------------------\n\n";
	exit(0);
}

if ($child > 0)
{
	echo "\n\n-----------------------Forked child, waiting for it to end-------------------------\n\n";
	pcntl_wait($status);
	echo "\n\n-----------------------Child stopped-------------------------\n\n";
}

```

**Expected behavior**
The script should end after 1 second. Instead, it goes on indefinitely because the child process never stops.

**Logs**
With the log level set to trace, a normal process & forked process gives the same output:
```
-18 01:19:33.127062+00:00 [PID: 12903] [TID: 12903] [DEBUG]    [Ext-API] [elastic_apm_API.c:229] [elasticApmSendToServer] Exiting...; resultCode: resultSuccess (0); 
2022-02-18 01:19:33.127099+00:00 [PID: 12903] [TID: 12903] [DEBUG]    [Bootstrap] [BootstrapStageLogger.php:215] [shutdown] Successfully completed shutdown sequence
2022-02-18 01:19:33.127153+00:00 [PID: 12903] [TID: 12903] [DEBUG]    [Util] [util_for_PHP.c:154] [callPhpFunction] Exiting...; resultCode: resultSuccess (0); 
2022-02-18 01:19:33.127172+00:00 [PID: 12903] [TID: 12903] [DEBUG]    [C-to-PHP] [tracer_PHP_part.c:110] [shutdownTracerPhpPart] Exiting...; resultCode: resultSuccess (0); 
2022-02-18 01:19:33.127206+00:00 [PID: 12903] [TID: 12903] [DEBUG]    [Lifecycle] [lifecycle.c:351] [elasticApmRequestShutdown] Exiting...; resultCode: resultSuccess (0);
```

**Comment by thomas-thorburn-connect** [comment]
+1 Seeing this behaviour as well

**Comment by Rudolph-B** [comment]
Having a similar issue were forked children don't stop even after calling something explicitly such as die()

**Comment by waynegemmell** [comment]
We're having the same issues and have lodged a ticket with support.

**Comment by SergeyKleyman** [comment]
Fixed in [v1.5.1](https://github.com/elastic/apm-agent-php/releases/tag/v1.5.1)

**Comment by waynegemmell** [comment]
We've tested the fix. Looks like it's working as expected. Thanks.
